files/build_templates/per_namespace/swss.service.j2

```diff
@@ -0,0 +1,36 @@
+[Unit]
+Description=switch state service
+{% if multi_instance == 'true' %}
+Requires=database@%i.service
+After=database@%i.service
+    {% if sonic_asic_platform == 'vs' %}
```

Comment:
Do these leading spaces get written to the generated file? If so, you can still get some visual indenting without adding spaces to the generated file by adding the spaces inside the brackets as follows, I believe:

```
{%     if sonic_asic_platform == 'vs' %}
```

Reply:
Fixed as per comment.